### ISSUE TITLE
Receiving Error: 'spaceeye.split' does not exist in the workflow graph when running visualize dataset notebook of "Crop Segmentation"

### ISSUE THREAD

**Issue by saqibzia-dev** [microsoft/farmvibes-ai#35]
Hi,
    I am running **visualize dataset** notebook of **crop segmentation** on Azure VM with recommended specs,I didnot made any changes and running the default code provided by you guys but I am receiving error when I run the workflow.
The error I receive is: 
    "Unable to run workflow with provided parameters. Tried to connect port rasters from op spaceeye.split to port raster of op ndvi.compute_index, but 'spaceeye.split' does not exist in the workflow graph." 

I have attached the code and error for your reference.I am also using reverse proxy which is hidden in the below screenshots.

![1 code crop segmentation](https://user-images.githubusercontent.com/56071934/216236448-78eb6188-545e-4024-b908-2a0c382ddb08.png)
![2 error crop segmentation](https://user-images.githubusercontent.com/56071934/216236458-6240f22f-83f8-4ca1-929a-cdeabe4caa4b.png)

**Comment by rafaspadilha** [comment]
Hi, @saqibzia-dev. Thank you for identifying this issue. 

We have recently made a few changes in the names of sinks throughout SpaceEye workflows and forgot to update the custom-workflow yaml files in the crop segmentation examples. This will be fixed in the next release. 

In the meantime, you can run the visualization notebook by changing the content of `notebooks/crop_segmentation/wf_viz_crop_seg.yaml` to: 

```
name: visualize_crop_segmentation
sources:
  user_input:
    - s2.user_input
    - spaceeye.user_input
    - cdl.user_input
sinks:
  s2_rasters: s2.raster ### NEW SINK FOR VISUALIZATION
  s2_masks: s2.mask ### NEW SINK FOR VISUALIZATION
  se_rasters: spaceeye.raster ### NEW SINK FOR VISUALIZATION
  ndvi: ndvi.index_raster
  cdl: cdl.raster
parameters:
  pc_key:
tasks:
  s2: ### NEW TASK FOR VISUALIZATION
    workflow: data_ingestion/sentinel2/preprocess_s2_improved_masks
  spaceeye:
    workflow: data_ingestion/spaceeye/spaceeye_interpolation
    parameters:
      pc_key: "@from(pc_key)"
  ndvi:
    workflow: data_processing/index/index
    parameters:
      index: ndvi
  cdl:
    workflow: data_ingestion/cdl/download_cdl
edges:
  - origin: spaceeye.raster
    destination:
      - ndvi.raster

```

The only change from the previous version of the yaml is altering `spaceeye.split.rasters` to `spaceeye.rasters`

**Comment by rafaspadilha** [comment]
Marking this as closed by the fix in PR #44.
Let us know if you still find any issues while running.
